components/Hotspots/HotspotMapbox.js

```diff
@@ -1,13 +1,15 @@
 import { useState } from 'react'
 import { Tooltip } from 'antd'
-import { findBounds } from '../Txns/utils'
+import { findBounds, haversineDistance } from '../Txns/utils'
 import animalHash from 'angry-purple-tiger'
 import ReactMapboxGl, { Layer, Marker, Feature } from 'react-mapbox-gl'
 import { withRouter } from 'next/router'
 import ScaleControl from '../ScaleControl'
 import MapButton from './MapButton'
 import WitnessIcon from './WitnessIcon'
 
+const MAX_WITNESS_DISTANCE_THRESHOLD = 200
```

Comment:
I saw some discussion in discord deciding on a distance, however, I've seen presumably legitimate witnesses quite far (10-20km). What's the max theoretical distance? 

```suggestion
const MAX_WITNESS_DISTANCE_THRESHOLD = 50
```

Reply:
I think 200km is a good number for now :)